python/lsst/sims/GalSimInterface/galSimDetector.py

```diff
@@ -160,8 +160,8 @@ def _newOrigin(self, origin):
         @param [out] _newWcs is a WCS identical to self, but with the origin
         in pixel space moved to the specified origin
         """
-        _newWcs = GalSim_afw_TanSipWCS(self.detectorName, self.cameraWrapper, self.obs_metadata, self.epoch,
-                                       photParams=self.photParams, wcs=self._tanSipWcs)
+        _newWcs = GalSim_afw_TanSipWCS.__new__(GalSim_afw_TanSipWCS)
```

Comment:
@danielsf Could you have a look at the implications of this change please?   For my test runs, there seems to be a shift in the sky coordinate system by half a chip in both x and y, i.e., if I simulate the center sensor, R22_S11, instead of the visit pointing being located at the center of the chip, it's at the lower right corner (in non-CCS coordinates).
In any case, we need a fix here I think.

Reply:
I forgot to note that the images in both cases appear the same, it's just the coordinate system that's shifted.

Reply:
Strange.  I thought my change should have been exactly equivalent to what it had been doing.  Although I admittedly didn't try to follow the old code completely.  E.g. there are also things called `afw_crpix1` and `afw_crpix2`.  Maybe they also need to be updated?  Although they weren't before, so maybe not.

Reply:
Ah.  Very good.  Thanks!  I should have realized that was editing something in the original.

Done.